commons/commons-serialization-util/src/main/kotlin/uk/ac/ebi/serialization/common/FilesResolver.kt

```diff
@@ -15,23 +11,14 @@ class FilesResolver(private val basePath: File) {
         fileName: String,
     ): File {
         val name = fileName.replace("/", "-")
-        return baseFolder(subAccNo, version.toString()).createTempFile(name, ".json")
+        val baseFolder = basePath.resolve("$subAccNo/$version")
+        baseFolder.mkdirs()
+
+        return baseFolder.createTempFile(name, ".json")
     }
 
     fun createEmptyFile(fileName: String): File {
         val name = fileName.replace("/", "-")
-        return baseFolder("submissions").createTempFile(name, ".json")
-    }
-
-    private fun baseFolder(vararg keys: String): File {
-        val now = LocalDate.now()
-        val path =
-            basePath
-                .resolve(now.get(YEAR).toString())
-                .resolve(now.get(MONTH_OF_YEAR).toString())
-                .resolve(now.get(DAY_OF_MONTH).toString())
-                .resolveMany(*keys)
-        path.mkdirs()
-        return path
+        return Files.createTempFile(name, ".json").toFile()
```

Comment:
why we are migrating out of a deterministic folder and start using machine temp files?

Reply:
ahh got it, maybe them docunent this method is only for being use for random temporally json files and can be deleted without previous files.